### DISCUSSION TITLE
(Maybe) Found a way to force-reload System Settings Menu

### DISCUSSION THREAD

**Discussion by jtpotato** [FalconLee1011/Customized-Aerial-Screen-Saver#7] [General]
Not sure if this is a fluke or actually reproducible, but steps are as follows:
1. Edit `/Library/Application Support/com.apple.idleassetsd/Customer/entries.json` to remove everything from Customized Aerial Screen Saver
2. Restart system
3. Add new aerial through Customized Aerial Screen Saver
4. Restart system
5. It might appear after this point

**Comment by FalconLee1011** [comment]
Thanks for the help! this issue is fixed in [version 1.1](https://github.com/FalconLee1011/Customized-Aerial-Screen-Saver/commit/962b2bd481c2c9737451c555d3c4b7b4a8083cb4)!
